### ISSUE TITLE
`join()` don't accept more than two arguments

### ISSUE THREAD

**Issue by michalfita** [casey/just#1013]
```
target_json    := join("target/arch", ARCH, ARCH + ".json")
```
returns error:
```
error: Function `join` called with 3 arguments but takes 2
   |
34 | target_json    := join("kernel/arch", ARCH, ARCH + ".json")
   |                   ^^^^
```
while the documentation says:
> `join(a, b…)` - Join path `a` with path `b`. `join("foo/bar", "baz")` is `foo/bar/baz`. Accepts two or more arguments.

**Comment by casey** [comment]
Sorry for the confusion! This was a recent change, and hasn't made it into new version of `just` yet. I'll cut a new release and ping this issue when I do.

**Comment by michalfita** [comment]
Great. Thanks!
Waiting impatiently for this to happen!

**Comment by casey** [comment]
It is done! I just published 0.10.3 to crates.io, which includes variadic `join`, as well as a bunch of other new string manipulation functions. 

Binaries will appear on the release page as they're published from CI: https://github.com/casey/just/releases/tag/0.10.3
